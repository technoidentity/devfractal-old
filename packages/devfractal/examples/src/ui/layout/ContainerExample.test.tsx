import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Container } from './ContainerExample'

test('Container Example', () => {
  const component: ReactTestRenderer = renderer.create(<Container />)
  expect(component.toJSON()).toMatchSnapshot()
})
