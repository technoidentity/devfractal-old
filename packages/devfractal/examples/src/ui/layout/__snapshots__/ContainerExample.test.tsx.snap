// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Centered container
    </h1>
    <div
      className="container"
    >
      <div
        className="notification"
      >
        This container is centered on desktop.
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="is-fullwidth"
    >
      <h1
        className="title"
      >
        Fluid container
      </h1>
      <div
        className="container is-fluid"
      >
        <div
          className="notification"
        >
          This container is 
          <strong>
            fluid
          </strong>
          : it will have a 32px gap on either side, on any viewport size.
        </div>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="is-fullwidth"
    >
      <h1
        className="title"
      >
        Breakpoint containers
      </h1>
      <div
        className="container is-widescreen"
      >
        <div
          className="notification"
        >
          This container is 
          <strong>
            fullwidth
          </strong>
           
          <em>
            until
          </em>
           the
          <code>
            $widescreen
          </code>
           breakpoint.
        </div>
      </div>
    </div>
    <section
      className="section"
    >
      <div
        className="container is-fullhd"
      >
        <div
          className="notification"
        >
          This container is 
          <strong>
            fullwidth
          </strong>
           
          <em>
            until
          </em>
           the
          <code>
            $fullhd
          </code>
           breakpoint.
        </div>
      </div>
    </section>
  </section>,
]
`;
